pkg/config/config.go

```diff
@@ -45,17 +45,6 @@ var (
 		field.WithRequired(false),
 		field.WithDisplayName("CA Certificate Path"),
 	)
-	caCertField = field.StringField(
```

Comment:
lets rename the display name on the file to be CA Certificate. In the UI, might be confusing to have it say "path". We can keep the actual field name the same, just update the display name 

we should update the caCertPathField to required now that there is only 1 field (a cert is required right?)

Reply:
nvm I see it is not required